packages/tre/src/runtime/config/workerd.capnp

```diff
@@ -325,6 +329,14 @@ struct Worker {
       # R2 bucket and admin API bindings. Similar to KV namespaces, these turn operations into
       # HTTP requests aimed at the named service.
 
+      wrapped @14 :WrappedBinding;
```

Comment:
Can this file be retrieved from `workerd` somehow? in order to keep it in sync?

Reply:
Yeah, I think so. Not a blocker for this PR obviously, but might be nice as a fast follow.